app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/welcome/WikiEducationDashboardFragment.kt

```diff
@@ -87,7 +88,7 @@ class WikiEducationDashboardFragment : Fragment() {
     }
 
     private fun proceedToLogin(url: String) {
-        Toast.makeText(context, "Logged In", Toast.LENGTH_SHORT).show()
+        view?.let { Snackbar.make(it, "Logged In", Snackbar.LENGTH_SHORT).show() }
```

Comment:
Use existing utils method
'view?.showSnackbar(message)'
 instead of 
Snackbar.make(it1, message, Snackbar.LENGTH_LONG).show()

Add an extra parameter in ViewUtils if you need.